test/select_items_integration_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 Searis AS
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"context"
+	"testing"
+
+	clarify "github.com/clarify/clarify-go"
+	"github.com/clarify/clarify-go/fields"
+)
+
+func TestSelectItems(t *testing.T) {
+	ctx := context.Background()
+	creds := getDefaultCreds(t)
+	client := creds.Client(ctx)
+	prefix := createPrefix()
+	// ---
```

Comment:
We have no precedence for these separators in the code.

Can we use a blank line instead?